front/components/assistant/conversation/EditConversationTitleDialog.tsx

```diff
@@ -0,0 +1,99 @@
+import {
```

Comment:
do we really want a dialog to rename ? I think the current experience was better (the in place edit)

The button in the menu should maybe trigger the inline edit instead / just not be there (to see with ed)

Reply:
As seen together : DeleteConversationDialog is already handling too many things:
- type "all" or "selected" as props 
- different action depending on the type
- creating a lot of overhead in SideBarMenu

I think the actions should be scoped to the Dialog for more readability --> new refactor PR inc. with LeaveConversationDialog and DeleteConversationDialog